docs/userguide/entry_point.rst

```diff
@@ -14,7 +14,7 @@ Console Scripts
 ===============
 
 First consider an example without entry points. Imagine a package
-defined thus::
+defined thus:
```

Comment:
Perfect! I was just about to file a bug report exactly about the code blocks on this page :laughing: